eng/Signing.props

```diff
@@ -17,8 +17,8 @@
   -->
   <ItemGroup>    
     <!-- Exclude two manually zipped servicehub symbol nupkgs from being signed -->
-    <ItemsToSign Remove="$(ArtifactsNonShippingPackagesDir)Microsoft.CodeAnalysis.Remote.ServiceHub.CoreComponents.x64.Symbols.nupkg"/>
-    <ItemsToSign Remove="$(ArtifactsNonShippingPackagesDir)Microsoft.CodeAnalysis.Remote.ServiceHub.CoreComponents.arm64.Symbols.nupkg"/>
+    <ItemsToSign Remove="$(ArtifactsNonShippingPackagesDir)Microsoft.CodeAnalysis.Remote.ServiceHub.CoreComponents.x64.Symbols.*.nupkg"/>
```

Comment:
Shouldn't it be `{project-name}.{version}.Symbols.nupkg`?

Reply:
I followed the current pattern, e.g.
`Microsoft.CodeAnalysis.CodeStyle.Fixes.Symbols.4.13.0-1.24511.9.nupkg`

Reply:
Arcade is looking for *.symbols.nupkg here:

https://github.com/dotnet/arcade/blob/ad64a76fd00a641116e79f77d48b42c6e7ed9b0a/src/Microsoft.DotNet.Arcade.Sdk/tools/Publish.proj#L98-L99

wonder if the other symbol package we produce is broken

Reply:
Re `Microsoft.CodeAnalysis.CodeStyle.Fixes.Symbols.nupkg` -- are we building that? Why would code style fixes need a symbol package?

Reply:
NuGet symbol package naming is documented here: https://learn.microsoft.com/en-us/nuget/create-packages/symbol-packages

Reply:
Let's fix the naming in a follow up PR.